public/lang/babele/de/investigator.pathOfCthulhuAbilities.json

```diff
@@ -41,36 +41,36 @@
             "category": "Allgemein"
         },
         "Bargain": {
-            "name": "Feilschen",
-            "category": "Sozial"
+            "name": "Handel",
+            "category": "Zwischenmenschlich"
         },
         "Biology": {
             "name": "Biologie",
             "category": "Akademisch"
         },
         "Bureaucracy": {
             "name": "Bürokratie",
-            "category": "Sozial"
+            "category": "Zwischenmenschlich"
         },
         "Chemistry": {
             "name": "Chemie",
             "category": "Technisch"
         },
         "Conceal": {
-            "name": "Kaschieren",
+            "name": "Tarnung",
             "category": "Allgemein"
         },
         "Cop Talk": {
             "name": "Polizeijargon",
-            "category": "Sozial"
+            "category": "Zwischenmenschlich"
         },
         "Craft": {
             "name": "Handwerk",
             "category": "Technisch"
         },
         "Credit Rating": {
-            "name": "Finanzkraft",
```

Comment:
Finanzkraft ist the current CoC Term would stay with it also Bonität is not really correct as it mostly describes the likelyhood that somebody pays their debts. 
To be Fair: Even Finanzkraft does not cover the full extend of credit-rating but its IMHO the best compromise.